docs/_pages/web_client.md

```diff
@@ -426,8 +426,31 @@ not log anything as long as nothing goes wrong.
 You can adjust the log level by setting the `logLevel` option to any of the values found in the `LogLevel` top-level
 export.
 
-You can also capture the logs without writing them to stdout by setting the `logger` option. It should be set to a
-function that takes `fn(level: string, message: string)`.
+```javascript
+const fs = require('fs');
+const { WebClient, LogLevel } = require('@slack/client');
+
+// increased logging, great for debugging
+const web = new WebClient(token, { logLevel: LogLevel.DEBUG });
+```
+
+You can also capture the logs without writing them to stdout by setting the `logger` option. The option should be set
+to an object that has the following methods:
+
+**Logger**
+
+| Method       | Parameters        | Return type |
+|--------------|-------------------|-------------|
+| `setLevel()` | `level: LogLevel` | `void`      |
+| `setName()`  | `name: string`    | `void`      |
+| `debug()`    | `...msgs: any[]`  | `void`      |
+| `info()`     | `...msgs: any[]`  | `void`      |
+| `warn()`     | `...msgs: any[]`  | `void`      |
+| `error()`    | `...msgs: any[]`  | `void`      |
+
+
+**NOTE**: The option can also take a function with the following signature, but this usage is deprecated:
```

Comment:
Suggested edit: "While the usage of logging functions is deprecated, the `logger` option will still currently accept a function if the method signature matches `fn(level: string, message: string)`."

The main thing I want to change is for the deprecation part to be at the beginning of the note.